### PR TITLE
I'd like to submit my essay

### PR DESCRIPTION
**Pull request by dargmuesli**
Add requirements.txt

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+gdom
+jq
```
Comment (kmein) [inline requirements.txt:2]: AFAIK jq isn't called from Python right now

**Review by kmein — CHANGES_REQUESTED** [review]
Please remove jq from requirements.txt

File: requirements.txt (hunk 1 above)
Comment (dargmuesli) [reply requirements.txt:2]: gdom neither, what prevents us form moving both to `frontend.py`?

Comment (kmein) [reply requirements.txt:2]: We don't know how to call gdom as a library from Python yet.

Comment (dargmuesli) [reply requirements.txt:2]: Something like https://stackoverflow.com/a/89243/4682621?

Comment (kmein) [reply requirements.txt:2]: Would've hoped not to have to do it this way :/
Yeah but maybe that's the simplest way.

Comment (dargmuesli) [reply requirements.txt:2]: I'll give it a shot

**Pull request merged**